docs/source/routing/self-hosted/index.mdx

```diff
@@ -19,31 +18,31 @@ flowchart LR;
   class clients secondary;
 ```
 
-<Note>
+Each release of Apollo Router includes:
 
-Self-hosting the GraphOS Router requires a current GraphOS [Enterprise or Free plan](https://www.apollographql.com/pricing).
+- A binary
+- A container image
+- A Helm chart
 
-</Note>
+## Local binary
 
-## Downloading and installing a self-hosted router
+Running Apollo Router directly from its binary speeds up local development, and it enables embedded use cases where containers are unavailable. 
 
-Apollo provides the router as both a binary and as images to run in your containerized deployments.
+Follow the [quickstart](/graphos/routing/get-started) to run a router binary.
 
-### Router binary
+## Kubernetes via Helm
 
-Apollo provides a binary of the router for multiple platforms. You can download the router bundle, extract the router binary, and run it. 
+Helm is a package manager for Kubernetes. Apollo provides an application Helm chart with each release of Apollo Router in GitHub Container Registry. Since router v0.14.0, Apollo has released each router Helm chart as an Open Container Initiative (OCI) image in `oci://ghcr.io/apollographql/helm-charts/router`.
 
-To learn more, follow the steps in the [self-hosted router installation](/graphos/reference/router/self-hosted-install) reference to run a router binary with a sample supergraph schema.
+Follow the [Kubernetes deployment guide] to deploy the router with a Helm chart.
 
-### Router container image
+## Container
 
-Apollo provides container images of the router to deploy in self-hosted environments. The router images are hosted on GitHub in its container repository. Each router release includes both production and debug images. 
+For containerization without k8s or Helm, Apollo provides container images with each release of Apollo Router. The images are available in GitHub, downloadable from `ghcr.io/apollographql/router` and the [router repo](https://github.com/apollographql/router/pkgs/container/router). Both debug and production images are provided.
```

Comment:
Beforing using `k8s`, add it as an abbreviation after the first use of "Kubernetes" on the page. (In the paragraph above.)